install/0000_90_machine-api-operator_03_servicemonitor.yaml

```diff
@@ -22,4 +22,43 @@ spec:
   selector:
     matchLabels:
       k8s-app: machine-api-operator
-
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: openshift-machine-api
+  name: machine-api-controllers
+  labels:
+    k8s-app: controller
+spec:
+  namespaceSelector:
+    matchNames:
+      - openshift-machine-api
+  selector:
+    matchLabels:
+      k8s-app: controller
+  endpoints:
+  - port: m-metrics
```

Comment:
Let's make these port names more descriptive.  <container-name>-metrics would be preferable.

Reply:
There is a constraint on 15 characters on the name. I'll try to pick a better naming.

Reply:
```
machine-mtrc
nodelink-mtrc
mhc-mtrc
machineset-mtrc
```
The 15 character limit is a bit difficult, but the above are all less than 15 characters, not sure about them, but they work 🤔 Is `mtrc` obviously metric?